Makefile

```diff
@@ -1,185 +1,92 @@
 
-# define VERBOSE to see executed commands
 # default build configuration
-TARGET=hyped
-MAIN=run/main.cpp
-CROSS=0
-NOLINT=0
+# TARGET  - define the binary name to compile into
+# MAIN    - define application entry point
+# CROSS   - set to 1 to use a cross-compiler, to compile on a laptop and run on BBB
+# NOLINT  - set to 1 to prevent linting the code
+# VERBOSE - set to 1 to print all commands Makefile runs
```

Comment:
@mkristien did I break it with the comments after the variables? 😅

Reply:
Yes. So it was you...

With the same line comments `VERBOSE` is not `1` but `1<several spaces here>` which breaks the `if` statements later on.